src/App/Pages/Accounts/HomePageViewModel.cs

```diff
@@ -25,11 +27,12 @@ public class HomeViewModel : BaseViewModel
         private string _selectedEnvironmentName;
         private bool _isEmailEnabled;
         private bool _canLogin;
-        private IPlatformUtilsService _platformUtilsService;
-        private ILogger _logger;
-        private IEnvironmentService _environmentService;
-        private IAccountsManager _accountManager;
-        private IConfigService _configService;
+        private bool _displayEuEnvironment;
+        private readonly IPlatformUtilsService _platformUtilsService;
+        private readonly ILogger _logger;
+        private readonly IEnvironmentService _environmentService;
+        private readonly IAccountsManager _accountManager;
+        private readonly IConfigService _configService;
```

Comment:
⛏️  Since you're here could you move the services to the top with the other services so we maintain the format as in the other files?